core/server/common/src/main/java/alluxio/master/Master.java

```diff
@@ -24,7 +22,7 @@
 /**
  * This interface contains common operations for all masters.
  */
-public interface Master extends JournalCheckpointStreamable {
+public interface Master extends Iterable<JournalEntry> {
```

Comment:
OK. i have added a new interface JournalEntryIterable that does the same as iterable<JournalEntry>. 

I can think of other names that do not have iterable as its suffix:
JournalEntryStreamable
JournalStreamable
JournalCheckpointStreamable.

I didn't pick them because I feel it is strange hide the fact this interface is actually an interable.. anyhow, let me know whether you have other suggestions on the class naming.